docs/source/simulator_support.rst

```diff
@@ -71,11 +72,11 @@ Verilator
 
 .. warning::
 
-    Verilator is in the process of adding more functionality to its VPI interface, which is used by cocotb to access the design.
-    Therefore, Verilator support in cocotb is currently experimental.
-    Some features of cocotb may not work correctly or at all.
+    **cocotb supports Verilator 5.006 and later.**
```

Comment:
Change that to the latest Verilator release we're actually testing, i.e., 5.022 (we're currently still using a git commit before this release in CI, but should update to this release now)

Reply:
#3739 switched to the new release.

The question is what do we consider the minimum supported version. 5.006 still works, but doesn't have all of the VPI improvements from later releases.

Reply:
Users want guidance on what version they should use, and what they can expect from using that version. If we say we "support" 5.006 but then users just run into problems that we know have been fixed, then we're not providing a good service to those users.

Since Verilator is super easy to update, I find it friendlier to say "cocotb supports Verilator 5.022 and later". The huge majority of users will then just get that version and be happy. And users who really cannot use that version might either reach out to us, or give earlier versions a try and use it if it works for them.

We don't even have tests running for 5.006!